src/test/spec/runner/operation.rs

```diff
@@ -271,10 +271,12 @@ impl TestOperation for InsertMany {
         let result = collection
             .insert_many(self.documents.clone(), self.options.clone())
             .await?;
-        let mut ids = Document::new();
-        for (id, bson) in result.inserted_ids {
-            ids.insert(id.to_string(), bson);
-        }
+        let ids: HashMap<String, Bson> = result
```

Comment:
If you don't end up using `result` again, you can use `into_iter()` instead of `iter()` and avoid needing to clone `v`